File: Leap Year Calculator/leap_year_calc.py
def is_leap_year(year):
    if year % 4 != 0 : return False
    if year % 100 != 0 : return True
    if year % 400 != 0 : return False
    return True


# One-line version

def is_leap_year(year):
    return False if year % 4 != 0 else True if year % 100 != 0 else False if year % 400 != 0 else True

File: Leap Year Calculator/leap-year-calc.cs
bool IsLeapYear(int year)
{
    if (year % 4 != 0) return false;
    if (year % 100 != 0) return true;
    if (year % 400 != 0) return false;
    return true;
}


// One-line version

bool IsLeapYear(int year) => year % 4 != 0 ? false : year % 100 != 0 ? true : year % 400 != 0 ? false : true;